From e9f749e16e9d45f5388f10ed6b0c530e1a249928 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-1.5-e9f749e16e9-20220705T105435Z-avarab@gmail.com>
In-Reply-To: <cover-v4-0.5-00000000000-20220705T105435Z-avarab@gmail.com>
References: <cover-v3-0.4-00000000000-20220701T102506Z-avarab@gmail.com>
	<cover-v4-0.5-00000000000-20220705T105435Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 4 Jul 2022 10:45:18 +0200
Subject: [PATCH v4 1/5] .gitignore: ignore generated "/*.res", not "*.res"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Adjust the overly broad .gitignore rule added in ce39c2e04ce (Provide
a Windows version resource for the git executables., 2012-05-24), a
subsequent commit will add *.res files in contrib/.

It might be possible to further narrow this down to simply
"/git.res". Let's leave it at ignoring "/*.res", which as ce39c2e04ce
and the current Makefile shows is the only thing we remove with "make
clean".

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .gitignore | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index a4522157641..6089c7f06bb 100644
--- a/.gitignore
+++ b/.gitignore
@@ -225,7 +225,7 @@
 *.hcc
 *.obj
 *.lib
-*.res
+/*.res
 *.sln
 *.sp
 *.suo
-- 
2.37.0.903.g3f2fd1cdc74

